src/sass/main.scss

```diff
@@ -114,4 +114,8 @@ input[type="range"] {
 
 label {
     user-select: none;
+}
+
+.text-muted {
+	color: silver;
```

Comment:
@jonez1 Similar to first comment. Tabs vs spaces-

Reply:
@danniehansen Fixed in https://github.com/danniehansen/activity-timer/pull/30/commits/3237ec9a46a1aaa816a4de292cf97e7cadbb720c